sfepy/homogenization/coefs_base.py

```diff
@@ -626,7 +626,7 @@ def get_slice(n_eigs, nn):
                 ii0 = get_slice(n_eigs[0], nn)
                 ii1 = get_slice(-n_eigs[1], nn)
                 eigs = nm.concatenate((eigs[ii0], eigs[ii1]))
-                mtx_q = nm.concatenate((mtx_q[:,ii0], mtx_q[:,ii1]), 1) 
+                mtx_q = nm.concatenate((mtx_q[:,ii0], mtx_q[:,ii1]), 1)
         else:
```

Comment:
Make a clean-up commit for this if you want - it is not related to the commit message.